packages/keycloak-user-management/src/components/forms/UserForm/index.tsx

```diff
@@ -128,7 +128,9 @@ const UserForm: React.FC<UserFormProps> = (props: UserFormProps) => {
   return (
     <Row className="layout-content">
       {/** If email is provided render edit user otherwise add user */}
-      <h5 className="mb-3">{props.initialValues.email ? EDIT_USER : ADD_USER}</h5>
+      <h5 className="mb-3 header-title">
+        {props.initialValues.email ? `${EDIT_USER} | ${initialValues.username}` : ADD_USER}
```

Comment:
why not check for `initialValues.id` instead? because not all users have an email set

Reply:
Not sure why it was not handled with `id`,
Updated.